test/unit/spec/LoginRouter_spec.js

```diff
@@ -1399,6 +1397,11 @@ Expect.describe('LoginRouter', function() {
       Util.loadWellKnownAndKeysCache();
       const successSpy = jasmine.createSpy('successSpy');
 
+      // In this test the id token will be returned succesfully. It must pass all validation.
+      // Mock the date to 10 seconds after token was issued.
+      const AUTH_TIME = (1451606400) * 1000; // The time the "VALID_ID_TOKEN" was issued
```

Comment:
Does this mean whenever our mock token changes, we'll need to update this `AUTH_TIME` as well?

Reply:
yes the mock time must match the issued time of the token, or it will fail validation